navit/xmlconfig.c

```diff
@@ -334,7 +335,7 @@ static char *element_fixmes[]= {
 };
 
 static void initStatic(void) {
-    elements=g_new0(struct element_func, 45); //44 is a number of elements + ending NULL element
+    elements=g_new0(struct element_func, 46); //45 is a number of elements + ending NULL element
```

Comment:
After reading a bit more of the context, it seems that this is out of scope for this PR. There are so many magic values, so this would need a major refactor.

Reply:
Agree.